### PR TITLE
[INLONG-1808] Optimize document of DataProxy about monitor metric

### PR DESCRIPTION
**Pull request by luchunliang**
Fixes apache/incubator-inlong#1808

where *XYZ* should be replaced by the actual issue number.

### Motivation

*Explain here the context, and why you're making that change. What is the problem you're trying to solve.*

### Modifications

*Describe the modifications you've done.*

### Verifying this change

- [ ] Make sure that the change passes the CI checks.

*(Please pick either of the following options)*

This change is a trivial rework / code cleanup without any test coverage.

*(or)*

This change is already covered by existing tests, such as *(please describe tests)*.

*(or)*

This change added tests and can be verified as follows:

*(example:)*
  - *Added integration tests for end-to-end deployment with large payloads (10MB)*
  - *Extended integration test for recovery after broker failure*

### Documentation

  - Does this pull request introduce a new feature? (yes / no)
  - If yes, how is the feature documented? (not applicable / docs / JavaDocs / not documented)
  - If a feature is not applicable for documentation, explain why?
  - If a feature is not documented yet in this PR, please create a followup issue for adding the documentation

### CODE REVIEW

**Review by dockerzhang — APPROVED** [review]

File: docs/modules/dataproxy/architecture.md [hunk 1]
```diff
@@ -150,3 +150,51 @@ Sink configuration example and corresponding notes
...
+# 4、Monitor metrics configuration instructions
+
+  DataProxy provide monitor indicator based on JMX, user can implement the code that read the metrics and report to user-defined monitor system.
+Source-module and Sink-module can add monitor metric class that is the subclass of org.apache.inlong.commons.config.metrics.MetricItemSet, and register it to MBeanServer. User-defined plugin can get module metric with JMX, and report metric data to different monitor system.
+
+  User can describe the configuration in the file "common.properties ". For example:
+  
+	metricDomains=DataProxy
+	metricDomains.DataProxy.domainListeners=org.apache.inlong.dataproxy.metrics.prometheus.PrometheusMetricListener
+	metricDomains.DataProxy.snapshotInterval=60000
```
Comment (aloyszhang) [inline docs/modules/dataproxy/architecture.md:163]: It's better to cover this in a code block or shell script block

**Review by gosonzhang — APPROVED** [review]
+1

File: docs/modules/dataproxy/architecture.md [hunk 2]
```diff
@@ -150,3 +150,51 @@ Sink configuration example and corresponding notes
...
+
+  The field of MetricItemValue.dimensions has these key(The fields of DataProxyMetricItem defined by the Annotation "@Dimension"):
+
+	public String clusterId:			DataProxy cluster ID.
+	public String sourceId:			DataProxy source component name.
+	public String sourceDataId:		DataProxy source component data id, when source is a TCP source, it will be port number.
+	public String inlongGroupId:		Inlong data group ID.
+	public String inlongStreamId:		Inlong data stream ID.
+	public String sinkId:				DataProxy sink component name.
+	public String sinkDataId:			DataProxy sink component data id, when sink is a pulsar sink, it will be topic name.
```
Comment (aloyszhang) [inline docs/modules/dataproxy/architecture.md]: place these fields in a form is better, and we can remove 'public String'

Comment (luchunliang) [reply docs/modules/dataproxy/architecture.md]: fixed, remove 'public String' and format it.

File: docs/modules/dataproxy/architecture.md (hunk 1 above)
Comment (luchunliang) [reply docs/modules/dataproxy/architecture.md]: fixed， use shell script block.

**Pull request merged**